bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJetsToLL_M-10to50_TuneCP5_13TeV-madgraphMLM-pythia8/DYJetsToLL_M-10to50_TuneCP5_13TeV-madgraphMLM-pythia8_proc_card.dat

```diff
@@ -0,0 +1,20 @@
+import model sm-ckm_no_b_mass
+
+#define p = p b b~
+#define j = j b b~
```

Comment:
please apply the instructions in https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T